detquantlib/data/databases/detdatabase.py

```diff
@@ -649,16 +649,16 @@ def load_account_positions(
         Loads account positions from the database, over a user-defined range of trading dates.
 
         Args:
-            start_trading_date: Start trading date
-            end_trading_date: End trading date
+            start_trading_date: Start trading date.
+            end_trading_date: End trading date.
             columns: Requested database table columns. Set columns=["*"] (i.e. as list) to get
                 all columns.
 
         Returns:
-            Dataframe containing quarterly account positions
+            Dataframe containing quarter-hourly account positions.
```

Comment:
```suggestion
            Dataframe containing account positions.
```

No need to mention quarter-hourly, the frequency might change in the future.